pkgs/test/test/runner/runner_test.dart

```diff
@@ -91,6 +91,8 @@ Usage: pub run test [files or directories...]
     --pause-after-load            Pauses for debugging before any tests execute.
                                   Implies --concurrency=1 and --timeout=none.
                                   Currently only supported for browser tests.
```

Comment:
Should add to this help text that this flag implies `--debug`

Reply:
Updated